src/main/java/org/apache/commons/lang3/annotation/ToEqualsExclude.java

```diff
@@ -0,0 +1,15 @@
+package org.apache.commons.lang3.annotation;
```

Comment:
Just saw this, when I was about to apply the patch. Please put the annotations in the builder package. I know it is a common pattern to create a specific package for annotation, but it simply makes no sense. It's better to package stuff by their domains and bot by their nature. For example you would never create a package called "interface" and put all your interfaces their. Why should we do this for annotations? They are no special in any way :-)